### PR TITLE
Update error messages for `addt`

### LINKED ISSUE #175: Update error messages for `addt`

### PR DESCRIPTION
**Pull request by kngys**
Fixes #175

### CODE REVIEW

File: src/main/java/seedu/address/logic/commands/AddTransactionCommand.java [hunk 1]
```diff
@@ -67,6 +67,10 @@ public CommandResult execute(Model model) throws CommandException {
         }
 
         List<Person> lastShownList = model.getFilteredPersonList();
+        if (lastShownList.isEmpty()) {
+            throw new CommandException(String.format(Messages.MESSAGE_EMPTY_PERSON_LIST, COMMAND_WORD));
+        }
+
```
Comment (beginner-web-developer) [inline src/main/java/seedu/address/logic/commands/AddTransactionCommand.java:73]: could you change the other commands to use this check too? I think its good

**Comment by beginner-web-developer** [comment]
Also it would be good to update the test cases to have an empty person list test

File: src/main/java/seedu/address/logic/commands/AddTransactionCommand.java (hunk 1 above)
Comment (kngys) [reply src/main/java/seedu/address/logic/commands/AddTransactionCommand.java:73]: When I add this into other commands it causes some tests in LogicManagerTest to fail, so I think maybe it's better if whoever is responsible for the specific command to update? In case anything is changed by mistake

**Comment by kngys** [comment]
> Also it would be good to update the test cases to have an empty person list test

Okay I have added the test case.

File: src/test/java/seedu/address/logic/commands/AddTransactionCommandTest.java [hunk 2]
```diff
@@ -88,6 +89,23 @@ public void execute_unfilteredList_success() {
         assertCommandSuccess(addTransactionCommand, model, expectedMessage, expectedModel);
     }
 
+    @Test
```
Comment (beginner-web-developer) [inline src/test/java/seedu/address/logic/commands/AddTransactionCommandTest.java:92]: could you copy this test to the other commands as well? Then it would be good to merge

File: src/test/java/seedu/address/logic/LogicManagerTest.java [hunk 3]
```diff
@@ -62,7 +62,8 @@ public void execute_invalidCommandFormat_throwsParseException() {
     @Test
     public void execute_commandExecutionError_throwsCommandException() {
         String deleteCommand = "delete 9";
-        assertCommandException(deleteCommand, MESSAGE_INVALID_PERSON_DISPLAYED_INDEX);
+        String expectedString = String.format(MESSAGE_EMPTY_PERSON_LIST, "delete");
+        assertCommandException(deleteCommand, expectedString);
```
Comment (kngys) [inline src/test/java/seedu/address/logic/LogicManagerTest.java:66]: @beginner-web-developer just to double check this is testing for an empty list right

File: src/test/java/seedu/address/logic/commands/AddTransactionCommandTest.java (hunk 2 above)
Comment (kngys) [reply src/test/java/seedu/address/logic/commands/AddTransactionCommandTest.java:92]: Added! Please help to review again thanks

File: src/test/java/seedu/address/logic/LogicManagerTest.java (hunk 3 above)
Comment (beginner-web-developer) [reply src/test/java/seedu/address/logic/LogicManagerTest.java:66]: I think so
